pytree.py

```diff
@@ -54,96 +58,85 @@ def recurse_dir(self, directory, parent_node):
+                # this_node.metadata["size"] = os.path.getsize(root + "/" + file)
+                parent_node.children.append(this_node)
 
 
     def show_tree(self, root_node, this_prefix, is_last, is_base, num_dirs, num_files):
         tree_string = u""
         prefix = u""
         next_prefix = u""
         num_printed = 0
-        #This works in python2, not python3
-        #root_node.children.sort()
-        #root_node.children.sort(key=lambda x: x.name)
         num_to_print = len(root_node.children)
         next_is_last = False
-
         if len(root_node.children) > 0:
             num_dirs += 1
         else:
             num_files += 1
-            
+
         if not is_base:
             if is_last:
                 prefix = this_prefix + u"\u2514\u2500\u2500 "
-                #prefix = this_prefix + "`-- "
+                # prefix = this_prefix + "`-- "
             else:
                 prefix = this_prefix + u"\u251c\u2500\u2500 "
-                #prefix = this_prefix + "|-- "
+                # prefix = this_prefix + "|-- "
         tree_string += (prefix + root_node.name)
-
         for child in root_node.children:
             if num_printed == (num_to_print - 1):
                 next_is_last = True
-            
+
             if is_last:
                 next_prefix = this_prefix + u"    "
             else:
                 next_prefix = this_prefix + u"\u2502\u00A0\u00A0 "
-                #next_prefix = this_prefix + "|   "
+                # next_prefix = this_prefix + "|   "
 
             if is_base:
                 next_prefix = u"\n"
             appended_string, num_dirs, num_files = self.show_tree(child, next_prefix, \
-                    next_is_last, False, num_dirs, num_files)
+                next_is_last, False, num_dirs, num_files)
             tree_string += appended_string
             num_printed += 1
         return tree_string, num_dirs, num_files
 
     def sort_tree(self, node):
         sorted_children = []
-        #node.children.sort(key=lambda x: x.name)
+        # node.children.sort(key=lambda x: x.name)
         for child in node.children:
             new_index = 0
             for child2 in sorted_children:
-                if re.sub('^__','',child.name.upper()) > re.sub('^__','',child2.name.upper()):
+                if re.sub('^__', '', child.name.upper()) > re.sub('^__', '', child2.name.upper()):
                     new_index += 1
             
             sorted_children.insert(new_index, child)
             self.sort_tree(child)
-
         node.children = sorted_children
         
-def main(root,regex=None,collect_metadata=False):
+def main(root, regex=None, collect_metadata=False):
     dir_tree = Tree(root)
     print(dir_tree)
 
 if __name__ == '__main__':
     import os
-    #main(sys.argv[1])
-    
+    # main(sys.argv[1])
     start_dir = "."
-
     if len(sys.argv) > 1:
         start_dir = sys.argv[1]
     main(start_dir, collect_metadata=True)
-
-
```

Comment:
FYI that [`pep8`](https://pep8.readthedocs.io/en/1.7.0/index.html) (the tool you were getting the warnings from via Code Climate) is fine with a single empty between these...just not two.

https://www.python.org/dev/peps/pep-0008/#blank-lines

Not saying I feel as strongly as the PEP 8 authors do about this...but just pointing it out.